help.txt

```diff
@@ -1,4 +1,4 @@
-To work on the tests, open a browser and visit http://localhost:{{port}}.
+To work on the tests, open a browser and visit http://{{host}}:{{port}}.
```

Comment:
Do you think adding the defaults here, and then also how to override them, is a good idea?